src/BUILD

```diff
@@ -115,43 +115,56 @@ py_binary(
     deps = [":create_embedded_tools_lib"],
 )
 
+# The tools Bazel uses to compile Java.
+# TODO(6316): Gradually remove the targets here.
```

Comment:
nit: I haven't seen this syntax for annotating TODOs with GitHub issue numbers yet.